### ISSUE TITLE
Delay JS + Advanced Ads conflict: ads are not displayed

### ISSUE THREAD

**Issue by NataliaDrause** [wp-media/wp-rocket#4103]
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version - 3.9.0.4
- Used the search feature to ensure that the bug hasn’t been reported before - yes

**Describe the bug**
When _Delay JavaScript Execution_ option is enabled (with or without standard exclusions) the ads added with [Advanced Ads](https://wordpress.org/plugins/advanced-ads/) plugin are not displayed.

**To Reproduce**
1. Display ads with [Advanced Ads](https://wordpress.org/plugins/advanced-ads/) plugin on a page
2. Enable _Delay JavaScript Execution_
3. Open the page with ads and scroll or move the mouse
4. Ads will not be displayed
5. No console errors in the dev tools.

The following exclusions in the Delay JS resolve the issue:
```
/jquery-?[0-9.](.*)(.min|.slim|.slim.min)?.js
/wp-content/uploads/(.*).js
advadsCfpQueue
adsbygoogle
adservice.google
```

**Expected behavior**
Ads should be displayed after interaction on a page.

**Screenshots**
Video recording in the note in the ticket: https://secure.helpscout.net/conversation/1553687181/274476/

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by GeekPress** [comment]
As this exclusion is containing several files, we can't do an automatic exclusion. You should be added here: https://docs.wp-rocket.me/article/1560-delay-javascript-execution-compatibility-exclusions

**Comment by GeekPress** [comment]
@NataliaDrause Can you please also add this exclusion here? https://docs.wp-rocket.me/article/1560-delay-javascript-execution-compatibility-exclusions

Thanks 🙏

**Comment by NataliaDrause** [comment]
@GeekPress sure! The list is already added here: https://docs.wp-rocket.me/article/1560-delay-javascript-execution-compatibility-exclusions#advanced-ads

**Comment by piotrbak** [comment]
Fixed in the 3.13 backend

**Comment by piotrbak** [comment]
Handled in 3.13 using dynamic lists
